.eslintrc.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+    env: {
+        es6: true,
+        browser: true,
+        node: true,
+    },
+    extends: ['airbnb', 'standard', 'standard-preact', 'prettier'],
+    plugins: ['babel', 'import', 'jsx-a11y', 'prettier'],
+    parser: 'babel-eslint',
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+        ecmaFeatures: {
+            jsx: true
+        }
+    },
+    rules: {
+        'linebreak-style': 'off', // Неправильно работает в Windows.
+
+        'arrow-parens': 'off', // Несовместимо с prettier
+        'object-curly-newline': 'off', // Несовместимо с prettier
+        'no-mixed-operators': 'off', // Несовместимо с prettier
+        'arrow-body-style': 'off', // Это - не наш стиль?
+        'function-paren-newline': 'off', // Несовместимо с prettier
+        'no-plusplus': 'off',
+        'space-before-function-paren': 0, // Несовместимо с prettier
+
+        'max-len': ['error', 100, 2, { ignoreUrls: true, }], // airbnb позволяет некоторые пограничные случаи
+        'no-console': 'error', // airbnb использует предупреждение
+        'no-alert': 'error', // airbnb использует предупреждение
+        'no-mixed-spaces-and-tabs': 'error',
+
+        'no-param-reassign': 'off', // Это - не наш стиль?
+        "radix": "off", // parseInt, parseFloat и radix выключены. Мне это не нравится.
+
+        'react/require-default-props': 'off', // airbnb использует уведомление об ошибке
+        'react/forbid-prop-types': 'off', // airbnb использует уведомление об ошибке
+        'react/jsx-filename-extension': ['error', { extensions: ['.js'] }], // airbnb использует .jsx
```

Comment:
Тогда почему у нас файлы все еще .js и удалось закоммитить?)

Reply:
это правило разрешает писать jsx в js. отредактировал правило таким образом что бы писать можно было и в js и в jsx.
```suggestion
       'react/jsx-filename-extension': ['error', { extensions: ['.js', '.jsx'] }]
```

Reply:
Если получится запихнуть в jsx, давай будем писать в jsx и запретим в js

Reply:
сделано